### PR TITLE
Add atlas_doc_format

### PR DESCRIPTION
**Pull request by andymac4182**
Add https://developer.atlassian.com/cloud/jira/platform/apis/document/structure/ 
https://developer.atlassian.com/cloud/confluence/rest/v1/api-group-content/#api-wiki-rest-api-content-id-put

Adding the ADF format for APIs to read and write pages

Add `collectionName` which is required to insert images into ADF formatted pages for confluence

### CODE REVIEW

File: src/api/models/attachment.ts [hunk 1]
```diff
@@ -14,6 +14,7 @@ export interface Attachment {
     comment: string;
     mediaTypeDescription: string;
     fileId: string;
+    collectionName: string;
```
Comment (MrRefactoring) [inline src/api/models/attachment.ts:17]: Are you sure this extension always exists? Probably make sense to mark this property as optional?

Comment (andymac4182) [reply src/api/models/attachment.ts:17]: Will add as optional as not documented as always there.

**Comment by andymac4182** [comment]
I have updated this to be optional now.

**Comment by andymac4182** [comment]
Thanks for merging this 😀

**Pull request merged**